### PR TITLE
Improvement/apps 2071 authenticated paths

### PR DESCRIPTION
**Pull request by jokedurnez**

### CODE REVIEW

File: src/main/java/com/openlattice/chronicle/controllers/ChronicleStudyController.java [hunk 1]
```diff
@@ -186,7 +185,7 @@ public Integer updateAppsUsageAssociationData(
     }
 
     @RequestMapping(
-            path = PARTICIPANT_PATH + DATA_PATH + STUDY_ID_PATH + ENTITY_KEY_ID_PATH + PREPROCESSED_PATH,
+            path = PARTICIPANT_PATH + DATA_PATH + AUTHENTICATED + STUDY_ID_PATH + ENTITY_KEY_ID_PATH + PREPROCESSED_PATH,
             method = RequestMethod.GET,
             produces = { MediaType.APPLICATION_JSON_VALUE, CustomMediaType.TEXT_CSV_VALUE }
```
Comment (anzioka) [inline src/main/java/com/openlattice/chronicle/controllers/ChronicleStudyController.java:190]: shouldn't this start with `AUTHENTICATED`? It conflicts with chronicle api

File: src/main/java/com/openlattice/chronicle/controllers/ChronicleStudyController.java [hunk 2]
```diff
@@ -212,16 +211,13 @@ public Integer updateAppsUsageAssociationData(
...
+        String token = ( (JwtAuthentication) authentication ).getToken();
+        return chronicleService.getAllPreprocessedParticipantData( studyId, participantEntityKeyId, token );
 
-        throw new InsufficientAuthenticationException( "request is not authenticated" );
     }
 
     @RequestMapping(
-            path = PARTICIPANT_PATH + DATA_PATH + STUDY_ID_PATH + ENTITY_KEY_ID_PATH,
+            path = PARTICIPANT_PATH + DATA_PATH + AUTHENTICATED + STUDY_ID_PATH + ENTITY_KEY_ID_PATH,
             method = RequestMethod.GET,
```
Comment (anzioka) [inline src/main/java/com/openlattice/chronicle/controllers/ChronicleStudyController.java:221]: same comment as above

File: src/main/java/com/openlattice/chronicle/controllers/ChronicleStudyController.java [hunk 3]
```diff
@@ -246,16 +242,12 @@ public Integer updateAppsUsageAssociationData(
...
-
-        throw new InsufficientAuthenticationException( "request is not authenticated" );
+        String token = ( (JwtAuthentication) authentication ).getToken();
+        return chronicleService.getAllParticipantData( studyId, participantEntityKeyId, token );
     }
 
     @RequestMapping(
-            path = PARTICIPANT_PATH + DATA_PATH + STUDY_ID_PATH + ENTITY_KEY_ID_PATH + USAGE_PATH,
+            path = PARTICIPANT_PATH + DATA_PATH + AUTHENTICATED + STUDY_ID_PATH + ENTITY_KEY_ID_PATH + USAGE_PATH,
             method = RequestMethod.GET,
```
Comment (anzioka) [inline src/main/java/com/openlattice/chronicle/controllers/ChronicleStudyController.java:251]: this shouldn't be authenticated

File: src/main/java/com/openlattice/chronicle/controllers/ChronicleStudyController.java [hunk 4]
```diff
@@ -246,16 +242,12 @@ public Integer updateAppsUsageAssociationData(
...
             produces = { MediaType.APPLICATION_JSON_VALUE, CustomMediaType.TEXT_CSV_VALUE }
```
Comment (anzioka) [inline src/main/java/com/openlattice/chronicle/controllers/ChronicleStudyController.java:252]: this should also start with `AUTHENTICATED`

**Comment by jokedurnez** [comment]
develop doesn't have the paths for deletes yet @anzioka .  I was hoping to get this merged before the hard deletes, and add the paths in the hard-delete-branch since this PR much easier to review.

**Comment by jokedurnez** [comment]
merged into hard delete branch

**Pull request closed without merging**